test/e2e/actuals_test.go

```diff
@@ -202,21 +202,14 @@ func crpStatusUpdatedActual() func() error {
 	}
 }
 
-func workNamespaceAndDeploymentRemovedFromClusterActual(cluster *framework.Cluster) func() error {
+func workNamespaceRemovedFromClusterActual(cluster *framework.Cluster) func() error {
 	client := cluster.KubeClient
 
 	workNamespaceName := fmt.Sprintf(workNamespaceNameTemplate, GinkgoParallelProcess())
-	appConfigMapName := fmt.Sprintf(appConfigMapNameTemplate, GinkgoParallelProcess())
```

Comment:
The placement test uses the app config I believe?

Reply:
That logic is thus missing in many of the parts below I believe.

Reply:
app config will be created in the work namespace. So there is no need to check if app config is there if work namespace has been deleted.

Reply:
Oh, this is the removed actual. I read it wrong.